### PR TITLE
Helper texts & better page headings

### PR DESCRIPTION
**Pull request by Tschonti**
new page heading layout:
image

image

### CODE REVIEW

File: src/components/commons/PageHeading.tsx [hunk 1]
```diff
@@ -1,13 +1,35 @@
-import { Heading } from '@chakra-ui/react'
+import { Flex, Heading, useMediaQuery, VStack } from '@chakra-ui/react'
+import { ReactElement } from 'react'
 
 type Props = {
   title: string
+  children?: ReactElement
```
Comment (berenteb) [inline src/components/commons/PageHeading.tsx:6]: Extend PropsWithChildren

File: src/components/commons/PageHeading.tsx [hunk 2]
```diff
@@ -1,13 +1,35 @@
...
+      </Heading>
+    )
+  }
+  if (largeScreen) {
+    return (
+      <Flex my={3} justify="center" alignItems="center">
+        <span style={{ visibility: 'hidden', marginRight: 'auto' }}>{children}</span>
+
+        <Heading textAlign="center">{title}</Heading>
+        <span style={{ marginLeft: 'auto' }}>{children}</span>
```
Comment (berenteb) [inline src/components/commons/PageHeading.tsx:24]: Are you sure about duplicating children?

File: src/components/commons/PageHeading.tsx [hunk 3]
```diff
@@ -1,13 +1,35 @@
...
+      </Flex>
+    )
+  } else {
```
Comment (berenteb) [inline src/components/commons/PageHeading.tsx:27]: else is not needed here

**Review by berenteb — CHANGES_REQUESTED** [review]

File: src/components/commons/PageHeading.tsx (hunk 2 above)
Comment (Tschonti) [reply src/components/commons/PageHeading.tsx:24]: It was the easiest solution for keeping the title in the middle with a button to the right

**Review by kisbogdan-kolos — APPROVED** [review]

**Pull request merged**